docs/conf.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+#
+# ophyd documentation build configuration file, created by
+# sphinx-quickstart on Fri Nov  7 11:18:58 2014.
+#
+# This file is execfile()d with the current directory set to its containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+import ophyd
+
+# Add any Sphinx extension module names here, as strings. They can be extensions
+# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.doctest",
+    "sphinx.ext.autosummary",
+    "IPython.sphinxext.ipython_directive",
+    "IPython.sphinxext.ipython_console_highlighting",
+    "matplotlib.sphinxext.plot_directive",
+    "myst_parser",
+    "sphinx.ext.inheritance_diagram",
+    "numpydoc",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.intersphinx",
+    # For the card element
+    "sphinx_design",
+]
+
+# If true, Sphinx will warn about all references where the target cannot
+# be found.
+# nitpicky = True
+
+# The suffix of source filenames.
+source_suffix = ".rst"
+
+# The master toctree document.
+master_doc = "index"
+
+# General information about the project.
+project = "ophyd"
+copyright = "2014, Brookhaven National Lab"
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = ophyd.__version__
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = "sphinx"
+
+# -- Options for HTML output ---------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+html_theme = "pydata_sphinx_theme"
```

Comment:
Are we sure we don't need a package from https://pypi.org/project/pydata-sphinx-theme/?